components/FourYearPlanPage.tsx

```diff
@@ -390,8 +390,15 @@ export const FourYearPlanPage: FC<FourYearPlanType> = memo(
         if (credits <= 11) return warning.Low;
         else if (credits >= 19) return warning.High;
         else return null;
+      } else if (sem.season === season.Winter) {
+        if (credits <= 3) return warning.Low;
+        else if (credits >= 5) return warning.High;
+        else return null;
+      } else if (sem.season === season.Summer) {
+        if (credits <= 6) return warning.Low;
+        else if (credits >= 13) return warning.High;
+        else return null;
```

Comment:
https://www.uwstout.edu/academics/other-programs-courses/winterm-summer-registration
There does not appear to be any lower limits for winter/summer